src/node_contextify.cc

```diff
@@ -270,6 +271,83 @@ void ContextifyContext::IsContext(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void ContextifyContext::GetWrappedFunction(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Local<Context> context = env->context();
+  Isolate* isolate = env->isolate();
+
+  CHECK(args[0]->IsString());
+  Local<String> code = args[0].As<String>();
+
+  CHECK(args[1]->IsBoolean());
+  bool produce_cached_data = args[1]->IsTrue();
+
+  Local<Uint8Array> cached_data_buf;
+  if (!args[2]->IsUndefined()) {
+    CHECK(args[2]->IsUint8Array());
+    cached_data_buf = args[2].As<Uint8Array>();
+  }
+
+  // TODO(ryzokuken): Add support for sandboxing.
+
+  ScriptCompiler::CachedData* cached_data = nullptr;
+  if (!cached_data_buf.IsEmpty()) {
+    ArrayBuffer::Contents contents = cached_data_buf->Buffer()->GetContents();
+    uint8_t* data = static_cast<uint8_t*>(contents.Data());
+    cached_data = new ScriptCompiler::CachedData(
+      data + cached_data_buf->ByteOffset(), cached_data_buf->ByteLength());
+  }
+
+  ScriptCompiler::Source source(code, cached_data);
+
+  ScriptCompiler::CompileOptions options;
+  if (source.GetCachedData() == nullptr) {
+    options = ScriptCompiler::kNoCompileOptions;
```

Comment:
This option can also be set inside the if-block for `args[2]`.

Reply:
Same as above, will do.